### PR TITLE
Update FS API docs

### PR DESCRIPTION
**Pull request by calvinjia**
Update the FS API docs to use the new options type.

### CODE REVIEW

File: docs/en/api/FS-API.md [hunk 1]
```diff
@@ -11,18 +11,21 @@ can either use the [Alluxio Java Client](#Java-Client), or the
...
-A third option is to interact with Alluxio through its S3 API. Users can interact
+A fourth option is to interact with Alluxio through its S3 API. Users can interact
 using the same S3 clients used for AWS S3 operations. This makes it easy to change
 existing S3 workloads to use Alluxio.
 
 * Table of Contents
 {:toc}
 
-## Java Client
+# Java Client
```
Comment (ZacBlanco) [inline docs/en/api/FS-API.md]: I think throughout the docs we try to keep the highest level header within docs as h2 (`##`) I would consider keeping this and making the `Alluxio Java API` header h3

File: docs/en/api/FS-API.md [hunk 2]
```diff
@@ -68,8 +73,8 @@ users to specify non-default settings for the operation. For example:
 ```java
 FileSystem fs = FileSystem.Factory.get();
 AlluxioURI path = new AlluxioURI("/myFile");
-// Generate options to set a custom blocksize of 128 MB
-CreateFileOptions options = CreateFileOptions.defaults().setBlockSize(128 * Constants.MB);
+// Generate options to set a custom blocksize of 64 MB
+CreateFilePOptions options = CreateFilePOptions.newBuilder().setBlockSizeBytes(67108864).build();
```
Comment (ZacBlanco) [inline docs/en/api/FS-API.md]: Why not do `64 * Constants.MB`?

File: docs/en/api/FS-API.md (hunk 1 above)
Comment (calvinjia) [reply docs/en/api/FS-API.md]: Updated, thanks.

File: docs/en/api/FS-API.md (hunk 2 above)
Comment (calvinjia) [reply docs/en/api/FS-API.md]: Good call, updated.

**Pull request merged**